installer/tue-install-impl.bash

```diff
@@ -1019,3 +970,5 @@ then
         done
     fi
 fi
+
+return 0
```

Comment:
This is actually not required as nothing is checking this return value.

Reply:
Yep it is, in the tue-functions.bash

Reply:
But this return from `tue-install-impl` to `tue-install`. The return of `tue-install` is checked in `tue-functions`.
Am I right or is it something that I'm not seeing here?

Reply:
Yep, I have tested it and the return of `tue-install-impl` does end up in `tue-functions`. `tue-install` can't return as it is not sourced, but run in a child process.

Reply:
Alright.